Hi3Helper.Core/Lang/en.json

```diff
@@ -157,7 +157,7 @@
     "RepairBtn2FullDesc": "This method checks the file checksum/integrity, size and availability",
     "RepairBtn2Quick": "Quick Check",
     "RepairBtn2QuickDesc": "This method checks the file size and availability only. Please use Full Check if you want to check the integrity of the files",
```

Comment:
The thing why I put only "integrity" for the word is because it already includes the meaning of "the file should have not be corrupted". Not sure but I think we can leave it as is.

Reply:
Got it, the first sentence should be fine to be corrected then.

Reply:
Keeping this as is